docs/configuration.md

```diff
@@ -15,7 +17,7 @@ It's recommended that you also set either `EMPIRE_GITHUB_ORGANIZATION`, or `EMPI
 
 ### SAML Authentication
 
-Refer to the [docs](./saml) on configuring the SAML authentication backend.
+Refer to the [docs](./saml.md) on configuring the SAML authentication backend.
```

Comment:
To make this work with readthedocs, I thing this needs to remain `./saml`, but a section needs to be added to https://github.com/remind101/empire/blob/master/mkdocs.yml#L10